packages/docker-build/src/utils/exec-utils.ts

```diff
@@ -0,0 +1,20 @@
+// Copy from @yarnpkg/plugin-exec
+// https://github.com/yarnpkg/berry/blob/master/packages/plugin-exec/sources/execUtils.ts
```

Comment:
It's safer to add the commit hash to the URL.

```suggestion
// https://github.com/yarnpkg/berry/blob/63a77b5/packages/plugin-exec/sources/execUtils.ts
```